### ISSUE TITLE
Improve Smartmet Alert Client accessibility

### ISSUE THREAD

**Issue by geosaaga** [fmidev/weather-app#1190]
There are some date-time related issues with iOS VoiceOver.

- Short dates like "ke 28.1." should be "keskiviikko 28. tammikuu" for screen readers
- Date with time like "28.1. 08:45" should be "28. tammikuu 8:45" for screen readers
- vrk abbreviation should be "vuorokautta" for screen readers, check also other languages

Also case [ta] can be used for months in finnish language if easy to implement.

**Comment by pulakkam** [comment]
Closing this issue as the changes were included in the May release of Smartmet Alert Client and no feedback has been received since then. A new issue can be opened for any further accessibility improvements.
